integration-tests/bats/sql-merge.bats

```diff
@@ -19,6 +19,24 @@ teardown() {
     teardown_common
 }
 
+@test "sql-merge: DOLT_MERGE with displays hash." {
```

Comment:
I think we could probably safely skip this BATS tests if we could get the golang engine tests to check more of the results when a commit is returned. We could at least check that the commit hash is not empty and looks like a commit. With those results checks turned back on, I think we'd have enough coverage of merge results and wouldn't need this. Alternatively, instead of a brand new BATS test function, you could just drop a line or two into an existing merge BATS test to assert that the returned commit hash matches HEAD.